tests/lib/rules/no-warning-comments.js

```diff
@@ -244,6 +244,19 @@ ruleTester.run("no-warning-comments", rule, {
                     }
                 }
             ]
+        },
+        {
+            code: "// TODO: https://github.com/eslint/eslint/pull/13522#discussion_r470293411",
```

Comment:
:+1: nice self-explanatory test :)

Just, I think that in order to test that edge case, the long URL should be the first word, before `TODO:`. In this example, it will get `TODO:` as the first word.

Reply:
ohh shoot, yeah my bad.